### PR TITLE
Prepare for 1.3.0 release

### PR DESCRIPTION
**Pull request by KodrAus**
[Changeset since the last release](https://github.com/bitflags/bitflags/compare/1.2.1...master)

This includes over a year's worth of modernization and cleanups. The biggest external changes are the inclusion of new constant set operators to better support bitflags in const contexts (#244) and better support for multiple and empty `bitflags!` invocations (#235 and #225)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
+# 1.3.0
+
+- End empty doc comment with full stop ([#202])
```
Comment (konsumlamm) [inline CHANGELOG.md:3]:
```suggestion
- End `empty` doc comment with full stop ([#202])
```
To make it clear that we're talking about the function, not some "empty doc comment" (whatever that is, but I was confused at first).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+
+- Fix typo in crate root docs ([#206])
+
+- Document from_bits_unchecked unsafety ([#207])
+
+- Let `is_all()` ignore extra bits ([#211])
```
Comment (konsumlamm) [inline CHANGELOG.md:9]:
```suggestion
- Let `is_all` ignore extra bits ([#211])
```
For consistency.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
+# 1.3.0
+
```
Comment (konsumlamm) [inline CHANGELOG.md:2]:
```suggestion

- Add `#[repr(transparent)]` ([#187])

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+- Add named functions to perform set operations ([#244])
+
+- Fix typos in method docs ([#245])
+
+- Modernization of the `bitflags` macro to take advantage of newer features and 2018 idioms ([#246])
+
+- Fix regression (in an unreleased feature) and simplify tests ([#247])
+
+- Use `Self` and fix bug when overriding `stringify!` ([#249])
+
```
Comment (konsumlamm) [inline CHANGELOG.md:28]:
```suggestion

[#187]: https://github.com/bitflags/bitflags/pull/187
```

**Comment by KodrAus** [comment]
Thanks @konsumlamm! I think we should be ready to go now.

**Pull request merged**